audit/audit-template.yml

```diff
@@ -113,18 +114,57 @@ Resources:
             Principal:
               Service: "firehose.amazonaws.com"
             Action:
-              - 'kms:Decrypt'
-              - 'kms:Encrypt'
-              - 'kms:GenerateDataKey*'
-              - 'kms:DescribeKey'
-            Resource: '*'
+              - "kms:Decrypt"
+              - "kms:Encrypt"
+              - "kms:GenerateDataKey*"
+              - "kms:DescribeKey"
+            Resource: "*"
 
   FireHoseKMSKeyAlias:
     Type: AWS::KMS::Alias
     Properties:
       AliasName: !Sub "alias/${AWS::StackName}/${Environment}/FireHoseKMSKey"
       TargetKeyId: !Ref FireHoseKMSKey
 
+  AuditSqsKmsKey:
```

Comment:
As discussed in Slack, we need a more specific name for this key given that it's specifically being used for the queue that notifies us of files needing to be encrypted.

Reply:
I have renamed the queue and key